frontend/models/product-list/index.ts

```diff
@@ -48,20 +48,21 @@ export async function findProductList(
    filters: ProductListFiltersInput,
    deviceItemType: string | null = null
 ): Promise<ProductList | null> {
-   const deviceTitle = filters.deviceTitle?.eq ?? '';
+   const filterDeviceTitle = filters.deviceTitle?.eq ?? '';
```

Comment:
This just changed right?
```suggestion
   const filterDeviceTitle = filters.deviceTitle?.eqi ?? '';
```

Reply:
good catch thanks